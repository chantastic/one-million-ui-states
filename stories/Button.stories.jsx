import { Button } from "./Button";

export default {
  title: "Example/Button",
  component: Button,
  args: { primary: false, label: "Button" },
};

export const Primary = {
  args: {
    primary: true,
  },
};

export const Secondary = {};

export const Large = {
  args: {
    size: "large",
  },
};

export const Small = {
  args: {
    size: "small",
  },
};

// üìù Because these are just objects, we can compose stories from other stories
// export const LargePrimary = {
//   args: {
//     ...Large.args,
//     ...Primary.args,
//   },
// };
