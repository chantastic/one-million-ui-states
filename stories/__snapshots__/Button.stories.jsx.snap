// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example/Button Large smoke-test 1`] = `
Object {
  "checked": undefined,
  "children": Array [
    Object {
      "checked": undefined,
      "children": undefined,
      "name": "Button",
      "pressed": undefined,
      "role": "button",
      "value": undefined,
    },
  ],
  "name": "example-button--large",
  "pressed": undefined,
  "role": "WebArea",
  "value": undefined,
}
`;

exports[`Example/Button Secondary smoke-test 1`] = `
Object {
  "checked": undefined,
  "children": Array [
    Object {
      "checked": undefined,
      "children": undefined,
      "name": "Button",
      "pressed": undefined,
      "role": "button",
      "value": undefined,
    },
  ],
  "name": "example-button--secondary",
  "pressed": undefined,
  "role": "WebArea",
  "value": undefined,
}
`;

exports[`Example/Button Small smoke-test 1`] = `
Object {
  "checked": undefined,
  "children": Array [
    Object {
      "checked": undefined,
      "children": undefined,
      "name": "Button",
      "pressed": undefined,
      "role": "button",
      "value": undefined,
    },
  ],
  "name": "example-button--small",
  "pressed": undefined,
  "role": "WebArea",
  "value": undefined,
}
`;
